src/react/components/common/keyboardManager/keyboardRegistrationManager.test.ts

```diff
@@ -33,74 +33,85 @@ describe("Keyboard Registration Manager", () => {
-        keyboardManager.addHandler(KeyEventType.KeyPress, keyCode, handler3);
+        keyboardManager.addHandler(KeyEventType.KeyPress, keyCodes, handler1);
+        keyboardManager.addHandler(KeyEventType.KeyPress, keyCodes, handler2);
+        keyboardManager.addHandler(KeyEventType.KeyPress, keyCodes, handler3);
 
-        const keyDownHandlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCode);
+        const keyDownHandlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCodeString);
         expect(keyDownHandlers.length).toEqual(1);
 
-        const keyUpHandlers = keyboardManager.getHandlers(KeyEventType.KeyUp, keyCode);
+        const keyUpHandlers = keyboardManager.getHandlers(KeyEventType.KeyUp, keyCodeString);
         expect(keyUpHandlers.length).toEqual(2);
 
-        const keyPressHandlers = keyboardManager.getHandlers(KeyEventType.KeyPress, keyCode);
+        const keyPressHandlers = keyboardManager.getHandlers(KeyEventType.KeyPress, keyCodeString);
         expect(keyPressHandlers.length).toEqual(3);
     });
 
     it("can register multiple handlers for same key code", () => {
-        const keyCode = "Ctrl+H";
+        const keyCodeString = "Ctrl+H";
+        const keyCodes = [keyCodeString];
         const handler1 = (evt: KeyboardEvent) => null;
         const handler2 = (evt: KeyboardEvent) => null;
 
-        keyboardManager.addHandler(KeyEventType.KeyDown, keyCode, handler1);
-        keyboardManager.addHandler(KeyEventType.KeyDown, keyCode, handler2);
+        keyboardManager.addHandler(KeyEventType.KeyDown, keyCodes, handler1);
+        keyboardManager.addHandler(KeyEventType.KeyDown, keyCodes, handler2);
 
-        const handlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCode);
+        const handlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCodeString);
         expect(handlers.length).toEqual(2);
     });
 
     it("list of handlers cannot be mutated outside of API", () => {
-        const keyCode = "Ctrl+K";
+        const keyCodeString = "Ctrl+K";
+        const keyCodes = [keyCodeString];
         const handler1 = (evt: KeyboardEvent) => null;
 
-        keyboardManager.addHandler(KeyEventType.KeyDown, keyCode, handler1);
-        const handlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCode);
+        keyboardManager.addHandler(KeyEventType.KeyDown, keyCodes, handler1);
+        const handlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCodeString);
         const handlerCount = handlers.length;
 
         // Attempt to add more handlers
         handlers.push(handler1, handler1, handler1);
 
-        const newHandlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCode);
+        const newHandlers = keyboardManager.getHandlers(KeyEventType.KeyDown, keyCodeString);
         expect(newHandlers.length).toEqual(handlerCount);
     });
 
     it("can remove keyboard event handlers", () => {
```

Comment:
@wbreza does this not qualify as also a test for an array of keycodes? i thought about adding another test but it looks very similar to this